app/scripts/util.js

```diff
@@ -8,3 +8,5 @@ function getCookie(cname) {
   }
   return "";
 }
+
+var engine_url = "http://localhost:8081"
```

Comment:
i think we should not hardcode this. As the URL of engine is not the same as the origin of this file, and we decided not to reverse proxy through to the engine, we should deliver the base URL of engine via some API. What do you think?